src/internal/operators/max.ts

```diff
@@ -9,34 +9,36 @@ import { isFunction } from '../util/isFunction';
  * ![](max.png)
  *
  * ## Examples
+ *
  * Get the maximal value of a series of numbers
+ *
  * ```ts
- * import { of } from 'rxjs';
- * import { max } from 'rxjs/operators';
+ * import { of, max } from 'rxjs';
  *
- * of(5, 4, 7, 2, 8).pipe(
- *   max(),
- * )
- * .subscribe(x => console.log(x)); // -> 8
+ * of(5, 4, 7, 2, 8)
+ *   .pipe(max())
+ *   .subscribe(x => console.log(x));
+ *
+ * // Outputs
+ * // 8
  * ```
  *
  * Use a comparer function to get the maximal item
- * ```typescript
- * import { of } from 'rxjs';
- * import { max } from 'rxjs/operators';
- *
- * interface Person {
```

Comment:
TS can now infer type of objects passed to `of`, so I removed this interface entirely - I did this wherever I found similar interface declarations.